scrapyd-client/scrapyd-deploy

```diff
@@ -218,24 +223,24 @@ def _add_auth_header(request, target):
 
 def _http_post(request):
     try:
-        f = urllib2.urlopen(request)
+        f = urlopen(request)
         _log("Server response (%s):" % f.code)
-        print f.read()
+        print(f.read())
         return True
-    except urllib2.HTTPError, e:
+    except HTTPError as e:
         _log("Deploy failed (%s):" % e.code)
         resp = e.read()
         try:
             d = json.loads(resp)
```

Comment:
I'm not sure it works because it seems `resp` will be a bytes object, while in Python 3 json requires str.